smart-orchestrator-test/src_test/com/axonivy/utils/smart/orchestrator/demo/support/mock/request2.json

```diff
@@ -5,231 +5,8 @@
     "content" : "Instruction to follow:\n- Don't fill information related to approval\n[]\n"
   }, {
     "role" : "user",
-    "content" : "Computer is beeping after opening AxonIvy Portal. Need technical support."
+    "content" : "Computer is beeping after opening AxonIvy Portal\nYou must answer strictly in the following JSON format: {\n\"id\": (type: string),\n\"type\": (Type of the support ticket. It's an enum field; type: enum, must be one of [TECHNICAL, CUSTOMER, HR, OTHER]),\n\"name\": (Name of the support ticket. Must be start with prefix 'Support ticket: '; type: string),\n\"description\": (description of the support ticket; type: string),\n\"employeeUsername\": (username of the employee requested this ticket; type: string),\n\"firstApprover\": (username of the first approver of this ticket. Field type: approval; type: string),\n\"secondApprover\": (username of the second approver of this ticket. Field type: approval; type: string),\n\"aiApproval\": (approval decision by AI. Field type: approval; type: com.axonivy.utils.smart.orchestrator.demo.dto.AiApprovalDecision: {\n\"decision\": (Your decision. This is an enum field; type: enum, must be one of [APPROVE, REJECT, WARNING]),\n\"reason\": (The reason why you choose the decision. This field should be plain text.; type: string)\n}),\n\"firstApproval\": (approval decision by the first approver. Field type: approval; type: com.axonivy.utils.smart.orchestrator.demo.dto.ApprovalHistory: {\n\"approver\": (type: string),\n\"isApprove\": (type: boolean),\n\"comment\": (type: string)\n}),\n\"secondApproval\": (approval decision by the second approver. Field type: approval; type: com.axonivy.utils.smart.orchestrator.demo.dto.ApprovalHistory),\n\"requestedDate\": (The request date of this support ticket. Do not fill this field; type: string),\n\"requestor\": (type: com.axonivy.utils.smart.orchestrator.demo.dto.Employee: {\n\"username\": (Unique username of the employee; type: string),\n\"fullName\": (Full legal name of the employee; type: string),\n\"position\": (Position or job title of the employee within the organization; type: enum, must be one of [JUNIOR, SENIOR, MANAGER]),\n\"departmentId\": (Identifier of the department the employee belongs to; type: string),\n\"maxLeaveDays\": (Maximum number of leave days allocated to the employee per year; type: integer),\n\"usedLeaveDays\": (Number of leave days already used by the employee; type: integer),\n\"email\": (Work email address of the employee; type: string),\n\"department\": (Department details of the employee, ignored during JSON serialization; type: com.axonivy.utils.smart.orchestrator.demo.dto.Department: {\n\"id\": (type: string),\n\"name\": (type: string),\n\"firstLevelManager\": (type: string),\n\"secondLevelManager\": (type: string),\n\"firstLevelManagerEmp\": (type: com.axonivy.utils.smart.orchestrator.demo.dto.Employee),\n\"secondLevelManagerEmp\": (type: com.axonivy.utils.smart.orchestrator.demo.dto.Employee)\n})\n})\n}"
```

Comment:
looks as if we have lost json_schema structured output: modelBuilder.responseFormat("json_schema")